packages/frontend/src/config/addresses.ts

```diff
@@ -1,20 +1,20 @@
-import { addresses as mainnetAddresses, networks as mainnetNetworks } from './mainnet'
+import { mainnetAddresses, mainnetNetworks } from './mainnet'
 import { addresses as kovanAddresses, networks as kovanNetworks } from './kovan'
 import { addresses as goerliAddresses, networks as goerliNetworks } from './goerli'
 
 const reactAppNetwork = process.env.REACT_APP_NETWORK || 'mainnet'
-let network = reactAppNetwork
+let hopAppNetwork = reactAppNetwork
 if (reactAppNetwork === 'staging') {
-  network = 'mainnet'
+  hopAppNetwork = 'mainnet'
 }
 let addresses = kovanAddresses
 let networks = kovanNetworks
-const isMainnet = network === 'mainnet'
+const isMainnet = hopAppNetwork === 'mainnet'
```

Comment:
would it be worth using the chain slug enum here instead of string? (throughout this file where network is being assigned)